v2/terraform_flannel.py

```diff
@@ -328,3 +328,13 @@ def down(self):
             self.deployer.down()
         except Exception as e:
             raise e
+
+    def collectWindowsLogs(self):
+        self.opts.workspace = "/tmp/k8s-ovn-ovs"
+        collect_logs_script = os.path.join(self.opts.workspace, "v2", "collect-logs.ps1")
+        for vm_name in self.deployer.get_cluster_win_minion_vms_names():
+            logs_vm_path = os.path.join(self.opts.log_path, "%s.zip" % vm_name)
+            self.logging.info("Copying collect-logs script to node %s" % vm_name)
+            self._copyTo(collect_logs_script, "c:\\", vm_name, windows=True)
+            self._runRemoteCmd(("c:\\collect-logs.ps1 -Destination C:\\k"), vm_name, windows=True)
```

Comment:
I would prefer that collect-logs.ps1 gets Destination arg as a full path for the final zip containing the logs, rather than a dir. This way we don't have to make assumptions in this script regarding the final name of the logs zip. Less prone to errors that way.